packages/launchpad/src/global/GlobalEmpty.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <main class="text-center">
+    <h1 class="text-2rem mb-2">{{ t('globalPage.empty.title') }}</h1>
+    <p class="text-lg font-light text-gray-600 mb-6">{{ t('globalPage.empty.helper') }}</p>
+    <button type="button" @click="selectProject" class="min-w-220px relative block w-full border-2 bg-gray-50 border-gray-300 border-dashed rounded-lg p-12 text-center hover:border-gray-400 text-center">
+      <IconPlaceholder
+      class="-mb-8px mx-auto max-w-65px h-full relative justify-center w-full text-indigo-600"
```

Comment:
Is `"-mb-8px"` valid tailwind or is this `-` a typo?

Reply:
It's a negative bottom margin of 8px. It'll go away when we have icons that are sized appropriately. This one has awkward dimensions (weird extra whitespace around the actual icon).